kratos/includes/serializer.h

```diff
@@ -199,8 +199,8 @@ class KRATOS_API(KRATOS_CORE) Serializer
     {
         msRegisteredObjects.insert(RegisteredObjectsContainerType::value_type(rName,Create<TDataType>));
         msRegisteredObjectsName.insert(RegisteredObjectsNameContainerType::value_type(typeid(TDataType).name(), rName));
-	//std::cout<<" REGISTERED OBJECT "<<rName<<" TypeID "<<typeid (TDataType).name()<<std::endl;
-	//msRegisteredObjects.insert(RegisteredObjectsContainerType::value_type(rName,&pPrototype));
+    //std::cout<<" REGISTERED OBJECT "<<rName<<" TypeID "<<typeid (TDataType).name()<<std::endl;
```

Comment:
Leftover¿

Reply:
Nope, it was already there. My IDE probably made some whitespace adjustments while saving.